apps/antalmanac/src/components/Header/Header.tsx

```diff
@@ -51,6 +52,9 @@ const components = [
 
 const Header = ({ classes }: CustomAppBarProps) => {
     const isMobileScreen = useMediaQuery('(max-width:750px)');
+    const isDark = useThemeStore((store) => store.isDark);
+
+    const backgroundColor = isDark ? '#264a92' : undefined;
```

Comment:
Similar to the previous comment, prefer to store this variable somewhere. e.g.

```tsx
const DARK_HEADER_BACKGROUND = '#264a92'
```

Also, it looks like you opt for Hex and RGB interchangeably? I think it would be good to stick to one, and I think HSL would be a good option.